core/src/main/java/org/frankframework/http/HttpSender.java

```diff
@@ -209,12 +209,12 @@ protected HttpRequestBase getMethod(URI uri, Message message, ParameterValueList
 			case GET:
 				if (parameters!=null) {
 					queryParametersAppended = appendParameters(queryParametersAppended,relativePath,parameters);
-					if (log.isDebugEnabled()) log.debug("{}path after appending of parameters [{}]", getLogPrefix(), relativePath);
+					if (log.isDebugEnabled()) log.debug("path after appending of parameters [{}]", relativePath);
```

Comment:
```suggestion
					log.debug("path after appending of parameters [{}]", relativePath);
```

Reply:
There are roughly 350 occurences of such an if statement. Is it okay if I remove them in another PR, so the scope of this PR can stay relatively small?

Reply:
They should only be removed when using a lambda or when it's reference directly to a field (so not a method call).